ui/app/src/modules/Preview/PreviewConcierge.tsx

```diff
@@ -509,7 +509,7 @@ export function PreviewConcierge(props: any) {
             () => {
               issueDescriptorRequest({
                 site: siteId,
-                path,
+                path: path ?? models[parentModelId].craftercms.path,
```

Comment:
In **each** case, looks like there are several expressions to get the path. Can we consolidate?